docs/unity/features/memory-pressure.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Unity Memory Pressure
+description: Capture Unity Memory Pressure Statistics
+sidebar_position: 16
+---
+
+# Unity Memory Pressure Auto-Instrumentation
+
+## Overview
+
+The Unity SDK provides a simple system for capturing memory pressure statistics from the device at runtime.
+
+To enable, simply hit the checkbox for `Auto Instrumentation Memory Monitor` in your Unity SDK Settings Menu in Unity under `Scripting Define Symbols`. After hitting the checkbox, the object will be automatically generated at runtime. You will need to set custom thresholds for the object via the provided prefab in the package as well. And that's it! That's all there is to it!
```

Comment:
Can you add a screenshot showing the selection of this item in the settings menu?